README.md

```diff
@@ -111,9 +111,11 @@ Using Chocolatey has several advantages:
 Getting stuck is a natural part of development and happens to the best of us. It's better to reach out than to stay silent, and we encourage you to do so. If you find yourself stuck on a task for more than 45 minutes, here are a couple of places to get help:
 
 ##### Slack channels:
-1. **bwtc-technical-discussions** - Great place for technical questions that involve some code.
-2. **Your team channel** - When you start a project, you will be added to a private team channel. Feel free to ask your questions there! Your team lead may be able to help you through your blocker.
 
+
+1. **Your team channel** - When you start a project, you will be added to a private team channel. Feel free to ask your questions there! Your team lead may be able to help you through your blocker.
+2. **[bwtc-stackoverflow](https://stackoverflow.com/c/bwtc/questions)** - Did you know we have our own Stack Overflow? Take a look, it has some great quesions and answers. 
```

Comment:
Looks like this link implies a Slack channel called `bwtc-stackoverflow`, but I couldn't find one. The link links to our actual hub on StackOverflow's website. If that's the primary location to engage in discussion, should this be documented outside of the "Slack Channels" section of this readme?

Reply:
Totally agree, this should not be an implied slack channel. Im thinking of changing "Slack Channels" to "Communication Channels" to make this more broad.